tests/testdata/generate.rs

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env -S cargo -Z script
+---cargo
+[package]
+edition = "2021"
+[dependencies]
+rcgen = { version = "0.13", features = ["pem"] }
+---
+
+use std::fs::OpenOptions;
+use std::io::Write;
+use std::env;
+
+use rcgen::{DistinguishedName, KeyPair, DnType, IsCa, BasicConstraints};
+
+fn main() {
```

Comment:
Suggest changing this to `fn main() -> anyhow::Result<()>` and using `?` instead of `.unwrap()` all over the place.

Reply:
Done.